static/js/popovers.js

```diff
@@ -226,6 +249,29 @@ exports.register_click_handlers = function () {
         show_message_info_popover(this, rows.id(row));
     });
 
+    var emoji_list = emoji.emojis_by_name;
+
+     $("body").on("click", "#emoji_popover", function (e) {
```

Comment:
This might want to check that they clicked not just somewhere in the emoji popover but in particular on an emoji in it, in case we add something else like a heading to the popout later.